### ISSUE TITLE
In the current sample only the create command goes to the server

### ISSUE THREAD

**Issue by paulvanbladel** [TomasEkeli/ToDolittle#6]
The delete, mark as done, mark as not done are only handled locally in the front-end but not communicated to the server.

**Comment by TomasEkeli** [comment]
Left as an exercise for the reader, or is this something you think we should do to finish the sample @pavsaund?

**Comment by pavsaund** [comment]
yes, that's true - the current sample is "step 1" in hooking up the frontend to the create command in the backend. We never made it to "step 2" - to actually continue the process of wiring up specific commands for each action.
